### PR TITLE
Bump trie-db to 0.20.0

### PR DESCRIPTION
**Pull request by cecton**
- Prefix iterator. [#39](https://github.com/paritytech/trie/pull/39)

Related to #78

### CODE REVIEW

File: trie-db/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog].
 [Keep a Changelog]: http://keepachangelog.com/en/1.0.0/
 
 ## [Unreleased]
+
+## [0.20.0] - 2020-02-12
```
Comment (ordian) [inline trie-db/CHANGELOG.md]:
```suggestion
## [0.20.0] - 2020-02-07
```
Do you really want to publish it in 5 days :smile: ?

**Comment by cecton** [comment]
@ordian I have updated all the changelogs with their respective change which is usually an update of another crate. Is it good like that or should it be: `Update parity-util-mem to v0.5.1 [#78](https://github.com/paritytech/trie/pull/78)` everywhere instead?

File: trie-root/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "trie-root"
-version = "0.15.2"
+version = "0.16.0"
```
Comment (ordian) [inline trie-root/Cargo.toml]: i this this bump is not necessary as `reference-trie` is a dev-dependency, but let's be on the safer side

**Comment by cecton** [comment]
@ordian I don't know how to resolve this:

```
[0] [13:25:42] ~/r/t/t/reference-trie master > cargo publish --dry-run
    Updating crates.io index
   Packaging reference-trie v0.20.0 (/home/cecile/repos/trie/test-support/reference-trie)
   Verifying reference-trie v0.20.0 (/home/cecile/repos/trie/test-support/reference-trie)
error: failed to verify package tarball

Caused by:
  failed to select a version for the requirement `trie-db = "^0.20.0"`
  candidate versions found which didn't match: 0.19.2, 0.18.1, 0.18.0, ...
  location searched: crates.io index
required by package `reference-trie v0.20.0 (/home/cecile/repos/trie/target/package/reference-trie-0.20.0)`
[101] [13:25:49] ~/r/trie master > cd trie-db/
[0] [13:25:51] ~/r/t/trie-db master > cargo publish --dry-run
    Updating crates.io index
   Packaging trie-db v0.20.0 (/home/cecile/repos/trie/trie-db)
   Verifying trie-db v0.20.0 (/home/cecile/repos/trie/trie-db)
error: failed to verify package tarball

Caused by:
  failed to select a version for the requirement `reference-trie = "^0.20.0"`
  candidate versions found which didn't match: 0.18.0, 0.15.2, 0.12.2, ...
  location searched: crates.io index
required by package `trie-db v0.20.0 (/home/cecile/repos/trie/target/package/trie-db-0.20.0)`
```

File: trie-db/Cargo.toml [hunk 3]
```diff
@@ -16,13 +16,13 @@ rustc-hex = { version = "2.1.0", default-features = false, optional = true }
...
 [dev-dependencies]
 env_logger = "0.6"
-memory-db = { path = "../memory-db", version = "0.18.1" }
+memory-db = { path = "../memory-db", version = "0.19.0" }
 rand = { version = "0.7", default-features = false, features = ["small_rng"] }
-trie-root = { path = "../trie-root", version = "0.15.2"}
+trie-root = { path = "../trie-root", version = "0.16.0"}
 trie-standardmap = { path = "../test-support/trie-standardmap", version = "0.15.2" }
 keccak-hasher = { path = "../test-support/keccak-hasher", version = "0.15.2" }
 # DISABLE the following line when publishing until cyclic dependencies are resolved https://github.com/rust-lang/cargo/issues/4242
```
Comment (ordian) [inline trie-db/Cargo.toml]: ^^^

Comment (ordian) [reply trie-db/Cargo.toml]: (feel free to ping me on riot as well)

**Pull request merged**